source/security/firmware-update.rst

```diff
@@ -79,22 +103,24 @@ After gathering required firmware binaries, capsule image can be generated using
                             Output file for signed image
       -q, --quiet           without output messages or temp files
 
-  For example, the following command generates a capsule image (``FwuImage.bin``) containing an IFWI image (``sbl.bios.bin``), CSME image (``csme.bin``), CSME Firmware Update Driver (``csme_fw_update_driver.bin``) and container component TSN MAC address inside container IPFW (``tsnmacaddr.bin``) signed by key ``FirmwareUpdateTestKey_Priv_RSA2048.pem``::

```

Comment:
add example command for container component update as it gets removed in combination with Bios.